frontend/src/metabase/containers/ErrorPages.jsx

```diff
@@ -54,3 +55,14 @@ export const Archived = ({ entityName, linkTo }) => (
     />
   </ErrorPageRoot>
 );
+
+export const SmallGenericError = ({
```

Comment:
Added a smaller generic error component because the other one was designed to be a whole page, and was too large for many individual components.